src/runtime/methodbinder.cs

```diff
@@ -13,19 +15,31 @@ namespace Python.Runtime
     /// </summary>
     internal class MethodBinder
     {
-        public ArrayList list;
+        public List<MethodBase> list;
         public MethodBase[] methods;
```

Comment:
Is there a reason to have both `list` and `methods`?

Reply:
I just change `ArrayList` to `List<MethodBase>`, the `methods` is the original member for cache in `GetMethods`.

Reply:
Fair enough